industry_restaurant/data/pos_config.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_config_main_seven_star_restaurant" model="pos.config">
+        <field name="name">Seven Star Restuarant - Sans Francisco</field>
+        <field name="use_pricelist" eval="1"/>
+        <field name="ship_later" eval="1"/>
+        <field name="cash_rounding" eval="1"/>
+        <field name="start_category" eval="1"/>
+        <field name="iface_customer_facing_display_local" eval="1"/>
+        <field name="iface_splitbill" eval="1"/>
+        <field name="iface_printbill" eval="1"/>
+        <field name="iface_tipproduct" eval="1"/>
+        <field name="iface_big_scrollbars" eval="1"/>
+        <field name="iface_start_categ_id" ref="pos_category_2"/>
+        <field name="gift_card_settings">scan_use</field>
+        <field name="is_header_or_footer" eval="1"/>
+        <field name="receipt_footer">Thank You! Visit Again.</field>
+        <field name="module_pos_restaurant_appointment" eval="1"/>
```

Comment:
The module is in the manifest, it should be enough.
```suggestion
```

Reply:
After that also it is not enabled so for that reason I have added it here

Reply:
That is weird. Isn't it a setting/field to activate?

Reply:
we need tp add this field otherwise it is not activated

Reply:
I just tried on a fresh database, installed pos_restaurant_appointment, and was able to assign table ressources on appointment... :thinking:

Reply:
We need to enable this field as I have checked in main runbot 17 Branch and it is not enabled
Video Ref - https://app.screencastify.com/v3/watch/E3xwC49isiKQ40NZTaEP

Reply:
Aah, ok, for website ... Ok then :pray: